day-one/overview.mdx

```diff
@@ -1,5 +1,118 @@
+  </Card>
+  <Card title="Enforce Compliance" icon="shield-check">
+    Bake security controls and architectural standards into every new service
+  </Card>
+  <Card title="AI-Powered Acceleration" icon="microchip-ai">
+    Use natural language to create entities, relationships, and business logic using Jovu.
+  </Card>
+  <Card title="Multi-Cloud Ready" icon="cloud">
+    Generate services pre-configured for AWS, Azure, or GCP ecosystems with our plugins.
+  </Card>
+</CardGroup>
+
+## Core Day One Capabilities
+
+<AccordionGroup>
+  <Accordion title="Resource Scaffolding" icon="code-simple" href="/day-one/create-service">
+    **Generate complete backend services** in minutes with:
+    - Pre-configured Node.js or .NET foundations
+    - Auto-generated REST & GraphQL APIs
+    - Production-ready authentication
+    - Database-agnostic ORM layers
+  </Accordion>
+
+  <Accordion title="Data Modeling" icon="database" href="/day-one/entities">
+    **Visual entity relationship modeling** with:
+    - AI-assisted schema creation
+    - Automatic migration scripts
+    - Field-level permissions
+    - Cross-service relationships
+  </Accordion>
+
+    <Accordion title="Schema Upload" icon="file-import">
+    - Import existing database schemas
+    - Automatically generate entity models
+    - Map relationships and constraints
+    - Support for PostgreSQL, MySQL, MongoDB, and others
+    - Preserve existing database structure
+  </Accordion>
+
+    <Accordion title="Plugin Integration" icon="plug">
+    - Install plugins from the public catalog (Node.js and .NET Blueprints)
+    - Extend functionality with authentication providers
+    - Add custom business logic and workflows
+    - Implement organization-specific standards
+    - Use private plugins for internal use
+  </Accordion>
+
+  <Accordion title="CI/CD Integration" icon="git-alt" href="/day-one/git-flows">
+    **Enterprise-grade Git workflows** featuring:
+    - Automated PR generation
+    - Multi-repo strategies
+    - Audit-ready change tracking
+  </Accordion>
+
+  <Accordion title="AI Assistant" icon="robot" href="/day-one/jovu">
+    **Jovu AI Copilot** capabilities:
+    - Natural language to code generation
+    - Architectural pattern suggestions
+    - Technical debt detection
+    - Context-aware documentation
+  </Accordion>
+</AccordionGroup>
+
+## Strategic Benefits for Development Teams
+
+<Steps>
+  <Step title="Focus on Business Logic" icon="brain">
+    Developers spend a lot less time on infrastructure code vs business logic
+  </Step>
+  <Step title="Reduce Human Error" icon="xmark">
+    Eliminate configuration mistakes through standardized templates
+  </Step>
+  <Step title="Accelerate Onboarding" icon="users">
+    Deploy first PR in less than 24 hours using approved patterns
+  </Step>
+  <Step title="Enterprise Scalability" icon="layer-group">
+    Support 1000+ resources with consistent observability patterns
```

Comment:
what do you mean?

Reply:
Maybe I can phrase this differently, but I meant the Architecture Graph can help you see your resources in a lot of different ways even up to 1000s of resources to help you clearly see what's what and who's in charge of what. @yuval-hazaz

Reply:
skip the number and just talk about visual/graph view of resource and relations